_security/configuration/demo-configuration.md

```diff
@@ -67,6 +68,11 @@ export OPENSEARCH_INITIAL_ADMIN_PASSWORD=<custom-admin-password>
 ```
 {% include copy.html %}
 
+### Mac
+Once you run `docker-compose.up` the OpenSearch server will be running. 
```

Comment:
Doesn't this link only apply to OpenSearch Dashboards? Also, this port link is relevant to more than Mac, so I'm not sure why this is needed.

Reply:
Hi, @Naarcha-AWS , the issue raised by the user was that '20240417 | /docs/latest/security/configuration/demo-configuration/ | **Very confusing. Does not have instructions for Mac**; fixed by https://github.com/opensearch-project/documentation-website/pull/7381'

Something else was listed for every other section so to add in some information for Mac, I added this in - perhaps there is a better approach to address Mac users here? Might you have a suggestion please?

Reply:
I would keep it simple and steer Mac users towards the Docker instructions, since Mac users won't have to change those to use Mac. Maybe we can add a note to the top of the section that any instructions specific to Linux or Unix distributions will work on Mac?

Reply:
Thanks @Naarcha-AWS , I have updated the code - what do you think or will I add a separate note too?